sdcm/nemesis.py

```diff
@@ -3332,6 +3332,7 @@ def disrupt_run_unique_sequence(self):
         InfoEvent(message='Starting grow_shrink disruption').publish()
         self.disrupt_grow_shrink_cluster()
         InfoEvent(message="Finished grow_shrink disruption").publish()
+        time.sleep(5 * 60)  # sleeping 5 minutes to ensure latency spikes are related to end of test and not to nemesis
```

Comment:
isn't that a bit crude doing that always

maybe something like `nemesis_sequence_sleep_between_ops`, or even using it, seem like it only use in the perf with nemesis test

Reply:
I don't understand.
Where do you see the spike? After the last decommission? 
Why would it be related to the decommission?
How would sleeping at the end of the test help it?
The calculation of latency is between the the info events.

Also, as @fruch mentioned, if we decide to use the sleep there, just use the same "time.sleep(sleep_time_between_ops)".

Reply:
@roydahan , did you go through the issue listed in the commit message? if you did, you would know the answer for all the questions...
in that issue, @slivne asked to have a 5 min sleep... of course we can use `sleep_time_between_ops`, but in that case, it will be a 10 min sleep, but I'm fine with that

Reply:
and the latency calculation is not affected by the spike, so we are adding this wait here, to be sure if it happens because we started terminating resources, or if still happens very close to the last decommission, we will know we have an issue around there...

Reply:
Of course it happens because we’re stopping the stress. We don’t need this sleep to prove it and if we do, you can run it one time with a sleep and prove it.

Reply:
giving it a run [here](https://jenkins.scylladb.com/view/staging/job/scylla-staging/job/fabio/job/perf_with_nemesis_report_improvements/job/perf-sleep-before-end/2/)